src/wallet/db.h

```diff
@@ -142,6 +140,9 @@ class WalletDatabase
 
     virtual void ReloadDbEnv() = 0;
 
+    /** Return path to main database file for logs and error messages. */
+    virtual std::string Filename() = 0;
```

Comment:
There is a `m_file_path` variable in `WalletDatabase` that does this. If we add `Filename`, we should remove that.